src/types/spidUser.ts

```diff
@@ -0,0 +1,29 @@
+import * as t from "io-ts";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+//
+import { EmailAddress } from "../../generated/io-backend/EmailAddress";
+import { FiscalCode } from "../../generated/io-backend/FiscalCode";
+import { SpidLevel } from "../../generated/io-backend/SpidLevel";
+
+import { Issuer } from "./issuer";
+
+// required attributes
+export const SpidUser = t.intersection([
+  t.interface({
+    authnContextClassRef: SpidLevel,
+    email: EmailAddress,
+    familyName: t.string,
+    fiscalNumber: FiscalCode,
+    getAssertionXml: t.Function,
+    issuer: Issuer,
```

Comment:
there's no need to declare a separate type (unless you need it), you can inline the definition here:

```suggestion
    issuer: t.interface({ name: t.string }),
```